opencontrail/files/3.0/collector/contrail-alarm-gen.ini

```diff
@@ -0,0 +1,13 @@
+[program:contrail-alarm-gen]
+command=/usr/bin/contrail-alarm-gen -c /etc/contrail/contrail-alarm-gen.conf
+priority=440
+autostart=true
+killasgroup=true
+stopsignal=KILL
```

Comment:
This is config file from contrail-analytics package with one modification: -c /etc/contrail/contrail-alarm-gen.conf which is required to start alarm gen with correct address of Collector